Automattic/Simplecons
# Add Warning Icon (24px)

### Fix
Add a warning icon.  The design is based off the [`info`](https://github.com/Automattic/Simplecons/blob/master/SVG/24px/info.svg) icon with the `i` rotated 180 degrees.  See the screenshots below for illustration.

<kbd><a href="https://user-images.githubusercontent.com/3827611/96160952-e2bf2c80-0ed3-11eb-85db-3a4b612523c0.png"><img src="https://user-images.githubusercontent.com/3827611/96160952-e2bf2c80-0ed3-11eb-85db-3a4b612523c0.png" width="300"></a></kbd>

<kbd><a href="https://user-images.githubusercontent.com/3827611/96160985-ee125800-0ed3-11eb-8027-c1408c1baf9b.png"><img src="https://user-images.githubusercontent.com/3827611/96160985-ee125800-0ed3-11eb-8027-c1408c1baf9b.png" width="300"></a></kbd>

### Test
- Verify the icon appears as intended on a 24px grid.
- Verify the SVG code is consistent with other icons.
